pyomo/contrib/doe/doe.py

```diff
@@ -695,15 +498,12 @@ def _direct_kaug(self):
         var_dict = {}
         for name in list(self.param.keys()):
             # [0] is the scenario index
-            var_name.append(name+'[0]')
-            var_dict[name+'[0]'] = self.param[name]
+            var_name.append(name)
+            var_dict[name] = self.param[name]
```

Comment:
Is this copying self.param into var_dict? Likewise, is `var_name = list(self.param.keys())` equivalent?